app/css/index.css

```diff
@@ -1085,3 +1085,121 @@ video {
 		padding-top: 6rem;
 	}
 }
+
+.toggleSwitch span span {
```

Comment:
CSS is case-insensitive, prefer kebab-cased class names `.toggle-switch` over `.toggleSwitch`